### DISCUSSION TITLE
iCub VR Walking Platform

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#693] [Q&A] [answered]
Hi,

Is there any documentation available for the VR Walking Platform? I have only found information about the `Virtualizer Research & Development Kit` https://www.cyberith.com/products/. What other components would be necessary if we were to set this up ourselves?

![2024-03-20_15-38](https://github.com/robotology/community/assets/7019692/b9c93e28-eb23-4ddf-8051-8919f3c0c498)

**Comment by Nicogene** [comment]
cc @robotology/iit-artificial-mechanical-intelligence

**Comment by S-Dafarra** [comment, marked as answer]
Hi @jimingda, you can find information about our work on teleoperation at https://www.science.org/doi/10.1126/scirobotics.adh3834

There, we used iCub3, but the infrastructure is flexible to work with different robots.

To have the code installed, you can start by enabling the options ``ROBOTOLOGY_ENABLE_TELEOPERATION`` and ``ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS`` on the superbuild
